### DISCUSSION TITLE
如何在新的浏览器标签页打开一个路由

### DISCUSSION THREAD

**Discussion by bzb745300** [vbenjs/vue-vben-admin#3197] [Q&A]
左边菜单有N个菜单，选中某个我想在新的浏览器标签页打开而不是在项目里面的tab中打开页面？如何去实现它，求助

**Comment by DesignHhuang** [comment]
可以参考：
![image](https://github.com/vbenjs/vue-vben-admin/assets/3368397/c9e599ba-c90f-46ba-a517-cf68f2ad6475)
